SiouxParser.py

```diff
@@ -26,7 +26,7 @@ def prettify_string(string):
     Remove tabs and newlines from string.
```

Comment:
update docstring as well please :)

Reply:
I have no push access to the patch-5 branch, so can't change it anymore, I'm sorry.